src/components/GalleryBrick/index.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+import FontAwesome from 'react-fontawesome';
+import styles from './styles';
+
+class GalleryBrick extends Component {
+  constructor(props){
+    super(props);
+    this.state = { currentPage: 1 }
+  }
+
+  calculatePages(){
+    let { items , itemsPerPage } = this.props;
+    let pages = parseInt(items.length / itemsPerPage);
+
+    return (items.length % itemsPerPage > 0) ? pages + 1 : pages;
+  }
+
+  renderItems(item, key, s){
+    return (
+      <div style={ s.imgGallery }>
+        <img key={ key } style={ s.image} src={ item } />
```

Comment:
the idea of the gallery is to render any type of items, not only images. So, instead of harcoding the <img/> element, you should directly render the "item" that comes as a param